adapters/ix/ix.go

```diff
@@ -35,7 +40,23 @@ type indexParams struct {
 	SiteID string `json:"siteId"`
 }
 
+type ixBidResult struct {
+	Request      *callOneObject
+	StatusCode   int
+	ResponseBody string
+	Bid          *pbs.PBSBid
+	Error        error
+}
+
+type callOneObject struct {
+	requestJSON bytes.Buffer
+	width       uint64
+	height      uint64
+}
+
 func (a *IxAdapter) Call(ctx context.Context, req *pbs.PBSRequest, bidder *pbs.PBSBidder) (pbs.PBSBidSlice, error) {
+	var prioritizedRequests, requests []callOneObject
+
 	if req.App != nil {
```

Comment:
This was added back when initial support for mobile apps was added to PBS.  It shouldn't be a problem to leave it in as a redundant check to ensure app requests don't make it through.

Reply:
yeah, it's not a big deal to us. Dead code just makes it a bit harder to debug the next time anyone changes your adapter, because they'll have to spend the time confirming that this logic isn't run.

Reply:
@dbemiller can we go ahead and resolve this if there is no major concern?

Reply:
yeah, for sure.